pyrene/src/components/Container/Container.tsx

```diff
@@ -0,0 +1,154 @@
+import React, {
+  FunctionComponent, useEffect, useRef, MouseEvent, useReducer,
+} from 'react';
+import clsx from 'clsx';
+
+import styles from './container.css';
+
+/**
+ * Container contain content and actions about a single subject.
+ */
+
+export interface ContainerProps {
+  /**
+   * Whether the container is collapsible when the user clicks on the header.
+   */
+  collapsible?: boolean,
+  /**
+   * Whether to display the content when the component is first mounted.
+   */
+  defaultExpanded?: boolean,
+  /**
+   * Javascript event handler.
+   */
+  onChange?: (event: MouseEvent<HTMLDivElement>) => void,
+  /**
+   * Sets the content to be rendered inside the component.
+   */
+  renderCallback: () => JSX.Element,
+  /**
+   * Sets the title displayed in the header of the component.
+   */
+  title: string,
+}
+
+interface State {
+  contentHeight: null | number,
```

Comment:
maybe worth moving to a type, because also used [here](https://github.com/open-ch/pyrene/pull/26/files#diff-f3bd19d7822f64efe9ab3b3341869cbc25fadb3c4e45c04c34cf349a7074f2c7R62)

Reply:
or we make it optional